Darius-Ndubi/Fast-Food-Fast
# Ft admin see orders 160087939

#### What does this PR do?
  Shows the admin endpoint

#### Description of Task to be completed?
- Allow the admin to see all orders
- Admin able to accept or decline an order
- Admin able to mark orders as complete or not
- Links for the admin to add, edit, delete fast food items

#### How should this be manually tested?
- Fork the [repo](https://github.com/Darius-Ndubi/Fast-Food-Fast)
- git clone https://github.com/Darius-Ndubi/Fast-Food-Fast.git
- cd Fast-Food-Fast
- git checkout ft-admin-see-orders-160087939
- open admin_profile.html in a browser
- https://darius-ndubi.github.io/admin_profile.html

#### What are the relevant pivotal tracker stories?
[#160087939](https://www.pivotaltracker.com/story/show/160087939)
[#160087863](https://www.pivotaltracker.com/story/show/160087863)
[#160087855](https://www.pivotaltracker.com/story/show/160087855)
